### DISCUSSION TITLE
[Menu entry swapper] Add a ctrl+click configuration

### DISCUSSION THREAD

**Discussion by athompkins98** [runelite/runelite#14926] [Ideas]
I like having left click set as eat/equip and shift+click set as drop but I think having ctrl+click configurations would be a great addition to the plugin, as you could set it to something like 'use' and save many bones from being unnecessarily buried. I've seen several other people asking for this feature in other forums and overall I think it would be a very nice addition.

**Comment by Nightfirecat** [comment]
For the case of bones, you can use the "Bones" swap, or use left-click customization for whichever specific bone types you'd like to prevent bury from being the left-click option.
